built.html

```diff
@@ -0,0 +1,67 @@
+<!DOCTYPE html>
+<html xmlns="http://www.w3.org/1999/xhtml" xml:lang="en" lang="en">
+  <head>
+    <meta http-equiv="content-type" content="text/html; charset=utf-8" />
+    <title>Vellum Test</title>
+    
+    <link href="tests/lib/font-awesome.css" type="text/css" rel="stylesheet" />
+    <link href="tests/lib/font-commcare.css" type="text/css" rel="stylesheet" />
+    <link href="tests/lib/hqstyle-core.css" type="text/css" rel="stylesheet"></link>
+    
+    <!-- Mocha and test-only dependencies -->
+    <!-- todo: call from tests/main.js -->
+    <link href="node_modules/mocha/mocha.css" type="text/css" rel="stylesheet"></link>
+    <script>
+      window.useBuilt = true;
```

Comment:
It wouldn't properly load a file with query string. I got fed up with it and just copied over index.html. not the best solution, but i just wanted it to work